src/helpers/schema.sql

```diff
@@ -7,6 +7,7 @@ CREATE TABLE spaces (
   flagged INT NOT NULL DEFAULT '0',
   hibernated INT NOT NULL DEFAULT '0',
   turbo INT NOT NULL DEFAULT '0',
+  turbo_expiration INT NOT NULL DEFAULT '0',
```

Comment:
Sorry didn't notice it earlier, maybe we can use the same type as `created` and `updated`?

```sql
  created BIGINT NOT NULL,
  updated BIGINT NOT NULL,
 ```

Reply:
yeah, i didn't know int / bigint were not signed in sql. updated to use bigint so we can have 4 bytes of positive range : f655ce85ea0aa9cc64dd312ab7f0d2b0c4f2dc9e